app/js/actions/QuestionActions.js

```diff
@@ -22,6 +22,13 @@ QuestionActions = {
       data: data
     });
   },
+  recieveUserQuestions: data => {
+    waitForQuestionStore();
+    AppDispatcher.dispatch({
+      actionType: ZhishiConstants.RECEIVE_USER_QUESTIONS,
+      data: data
+    });
+  },
```

Comment:
Then propose the styling, so we use it as a convention. The point is so every code written feels uniform, as if by a single person. I like it though